weaver/store/mongodb.py

```diff
@@ -925,6 +925,7 @@ def find_jobs(self,
 
         search_filters.update(self._apply_status_filter(status))
         search_filters.update(self._apply_ref_or_type_filter(job_type, process, service))
+        search_filters.update(self._apply_tags_filter(tags, request))
```

Comment:
This seems to cause some problems too. `_apply_tags_filter` takes 1 argument, but there are two in this call here.